umap/static/umap/js/umap.controls.js

```diff
@@ -741,22 +741,50 @@ L.U.Map.include({
   _openFacet: function () {
     const container = L.DomUtil.create('div', 'umap-facet-search'),
       title = L.DomUtil.add('h3', 'umap-filter-title', container, L._('Facet search')),
-      keys = Object.keys(this.getFacetKeys())
+      facetKeys = this.getFacetKeys(),
+      keys = Object.keys(facetKeys)
 
-    const knownValues = {}
+    const facetCriteria = {}
 
     keys.forEach((key) => {
-      knownValues[key] = []
-      if (!this.facets[key]) this.facets[key] = []
+      if (facetKeys[key]["type"] === "date") {
+        if (!facetCriteria[key]) facetCriteria[key] = {
+	  "min": undefined,
```

Comment:
There are a couple of weird indentations, can you run PrettierJS against it?

Reply:
You are right, there are some tabs instead of spaces. Thanks for the hint, I will fix it.